src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -86,6 +86,17 @@ export interface ApplicationAttributes {
   deployments: Deployment[];
 }
 
+export interface ApplicationAttributesOverview {
+  appName: string;
+  deploymentsInfo: DeploymentNameAndVersion[];
+}
+
+export interface DeploymentNameAndVersion {
```

Comment:
Not a big fan of this name. It seems simultaneously over-descriptive and incomplete - rather than naming it after 2/3 of its properties, can we name it according to what it's used for perhaps?